assets/js/translator.js

```diff
@@ -730,6 +744,131 @@ function translateDoc() {
     }
 }
 
+window.onpopstate = function () {
+    // TODO: Could do the whole restoreChoices here? But would have to call the right translate() functions anyway
```

Comment:
Let's not leave TODO lying around. Let's create issues for them or just resolve them?